### PR TITLE
fix the styling

### PR DESCRIPTION
**Pull request by adescoteaux1**
# Description
[Link to Ticket](insert the link to your ticket inside the parenthesis here)
Please include a summary of the changes and the related issue. Please also
include relevant motivation, context, and images!

image
image

# How Has This Been Tested?
Please describe the tests that you manually ran to verify your changes (beyond any unit/integration tests written and ran).

## Screenshots
Please provide screenshots of manual testing (Scalar, frontend pages, etc.)

# Checklist

## General
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas

## Backend Changes (if applicable)
- [ ] I have written unit tests for my code
- [ ] I have written integration tests where appropriate
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have run make api-validate and make api-gen to keep the openAPI spec up to date
- [ ] I have tested error handling and edge cases
- [ ] Database migrations are included (if applicable)

## Frontend Changes (if applicable)
- [ ] I have tested in multiple browsers (Chrome, Firefox, Safari)
- [ ] Mobile responsive design verified (if applicable)
- [ ] I have run make generate-api to keep the API hooks up to date
- [ ] **For mobile features:** Tested on both iOS and Android devices
- [ ] **For mobile features:** Screenshots from both iOS and Android included below
- [ ] No console errors or warnings
- [ ] Loading states and error states handled

### iOS
[Add iOS screenshots here]

### Android
[Add Android screenshots here]

### CODE REVIEW

File: frontend/apps/mobile/app/(tabs)/index.tsx [hunk 1]
```diff
@@ -1,186 +1,368 @@
...
-      <ThemedText type="subtitle">
-        {item.event.title}
-      </ThemedText>
-
-      {item.event.description && (
-        <ThemedText style={styles.eventDescription}>
-          {item.event.description}
-        </ThemedText>
-      )}
+  console.log("[EventOccurrencesList] total events:", response.length);
```
Comment (josh-torre) [inline frontend/apps/mobile/app/(tabs)/index.tsx]: Please remove the console logs

File: frontend/apps/mobile/app.json [hunk 2]
```diff
@@ -12,15 +12,21 @@
       "supportsTablet": true,
       "infoPlist": {
         "NSLocationWhenInUseUsageDescription": "SkillSpark needs your location to show events and activities near you."
-      }
+      },
+      "config": {},
+      "bundleIdentifier": "com.adescoteaux.SkillSpark"
```
Comment (josh-torre) [inline frontend/apps/mobile/app.json]: hmmmm

File: frontend/apps/mobile/app.json [hunk 3]
```diff
@@ -12,15 +12,21 @@
...
         "backgroundColor": "#E6F4FE",
         "image": "./assets/images/generate.png"
       },
       "edgeToEdgeEnabled": true,
-      "predictiveBackGestureEnabled": false
+      "predictiveBackGestureEnabled": false,
+      "config": {
+        "googleMaps": {}
+      },
+      "package": "com.adescoteaux.SkillSpark"
```
Comment (josh-torre) [inline frontend/apps/mobile/app.json]: hmmmmm
